src/app.js

```diff
@@ -57,11 +58,13 @@ APP.use('/books', booksRouter);
 APP.use('/songs', songsRouter);
 APP.use('/logs', logsRouter);
 APP.use('/customers', customersRouter);
+APP.use('/avatar', avatarsRouter);
+
+APP.use(express.static('resources'));
```

Comment:
There is a better way to do the same which doesn't require a new module file
```suggestion
APP.use('/avatar',  express.static('resources/avatars'));
```
This is going to deliver the SVG files statically since the `avatars` directory, so we don't need a GET endpoint